velox/docs/functions/spark/misc.rst

```diff
@@ -15,6 +15,16 @@ Miscellaneous Functions
         SELECT at_least_n_non_nulls(2, 0, 1.0, NULL);  -- true
         SELECT at_least_n_non_nulls(2, 0, array(NULL), NULL);  -- true
 
+.. spark:function:: get_struct_field(struct, ordinal) -> T
+
+    Returns the value of nested subfield at position ``ordinal`` in the input ``struct``.
+    The input must be of row type and nested complex type is allowed.
+    The ``ordinal`` is 0-based, and if ``ordinal`` is negative or greater than or equal to
+    the children size of ``struct``, exception is thrown. ::
+
+        SELECT from_json('{"a": 1, "b": [1, 2, 3]}', 'a INT, b ARRAY<INT>').a; -- 1
```

Comment:
I notice below exception when running this sql in Spark, would you help confirm?

> error: ')' expected but string literal found.

Reply:
you should use three `"` instead of two.

```
sql("""SELECT from_json('{"a": 1, "b": [1, 2, 3]}', 'a INT, b ARRAY<INT>').a""").collect.foreach(println)
```

Reply:
also `set spark.sql.optimizer.excludedRules=org.apache.spark.sql.catalyst.optimizer.ConstantFolding` to make sure it's not optimized by spark and offload it to velox